templates/CRM/Cdntaxreceipts/Form/Settings.tpl

```diff
@@ -61,6 +61,11 @@
         <td class="content">{$form.receipt_prefix.html}
           <p class="description">{ts domain='org.civicrm.cdntaxreceipts'}Receipt numbers are formed by appending the CiviCRM Contribution ID to this prefix. Receipt numbers must be unique within your organization. If you also issue tax receipts using another system, you can use the prefix to ensure uniqueness (e.g. enter 'WEB-' here so all receipts issued through CiviCRM are WEB-00000001, WEB-00000002, etc.){/ts}</p></td>
       </tr>
+      <tr>
+        <td class="label">{$form.receipt_filename.label}</td>
+        <td class="content">{$form.receipt_filename.html}
+          <p class="description">{ts domain='org.civicrm.cdntaxreceipts'}Set default filename for receipt on download{/ts}</p></td>
```

Comment:
Let's modify this a bit to indicate that the file name we are changing will be a prefix. So something along the lines of "Set default file name for the PDF receipt. Note the file name set above is a prefix followed by the receipt number and possibly the donor or organization name. Eg. Receipt-112293James.pdf"